### DISCUSSION TITLE
What (minimum) version of Windows is required?

### DISCUSSION THREAD

**Discussion by SingWithMarv** [jamulussoftware/jamulus#2955] [Technical Support & Help] [answered]
Hello -- There is no Windows version requirement stated here [https://jamulus.io/wiki/Installation-for-Windows] for Jamulus, and I can't find it anywhere on Jamulus.io. Is it Windows 10 for 3.9.1? Or, like Mac, are there legacy versions for older versions of Windows? Thanks.

**Comment by ann0see** [comment, marked as answer]
Hi @SingWithMarv 
Thanks for the questions. Yes, officially we only support Windows 10 and 11. 

You can still install Jamulus on older Windows versions (down to Windows 7, I think), but it is strongly discouraged to do so and might not work in future.

**Comment by ann0see** [comment]
Just out of curiosity: Do you require support of an old version of Windows? What's your use case? Can you upgrade the PC or use a different OS? Even Windows 8.1 support by Microsoft is ending soon (or has already ended).

**Comment by SingWithMarv** [comment]
Thanks.

On Thu, Nov 3, 2022 at 2:47 PM ann0see ***@***.***> wrote:

> Hi @SingWithMarv <https://github.com/SingWithMarv>
> Thanks for the questions. Yes, officially we only support Windows 10 and
> 11.
>
> You can still install Jamulus on older Windows versions (down to Windows
> 7, I think), but it is strongly discouraged to do so and might not work in
> future.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/jamulussoftware/discussions/2955#discussioncomment-4051475>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A37TC75CMNCGTRHKIFXOR7LWGQXF5ANCNFSM6AAAAAARWM2EA4>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***
> com>
>


-- 

Marv Zauderer
Community Songleader & Enthusiator
SingWithMarv.com
San Anselmo Singing <http://centralmarinsingers.com> | Pt. Reyes Singing
<http://westmarinsingers.com> | Online Singing
<http://singonline.org/song-circle> | Insta
<http://instagram.com/singwithmarv> | Facebook
<https://www.facebook.com/singwithmarv>
Pronouns: he/him
*"Let your heart sing!"*

**Comment by SingWithMarv** [comment]
I had someone ask me if it works with 8.1, although they told me they plan
to upgrade shortly.

On Thu, Nov 3, 2022 at 2:49 PM ann0see ***@***.***> wrote:

> Just out of curiosity: Do you require support of an old version of
> Windows? What's your use case? Can you upgrade the PC or use a different
> OS? Even Windows 8.1 support by Microsoft is ending soon (or has already
> ended).
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/jamulussoftware/discussions/2955#discussioncomment-4051496>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A37TC77E7COHBNGNL7T5JZTWGQXP7ANCNFSM6AAAAAARWM2EA4>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***
> com>
>


-- 

Marv Zauderer
Community Songleader & Enthusiator
SingWithMarv.com
San Anselmo Singing <http://centralmarinsingers.com> | Pt. Reyes Singing
<http://westmarinsingers.com> | Online Singing
<http://singonline.org/song-circle> | Insta
<http://instagram.com/singwithmarv> | Facebook
<https://www.facebook.com/singwithmarv>
Pronouns: he/him
*"Let your heart sing!"*

**Reply by ann0see** [reply]
Ok. Good to know. The problem is that we need to move up to Qt6 in future. This means, Windows 8.1 and 7 will no longer be supported, and we're not sure if providing a legacy Windows build too is the right way to go if there is no demand.

**Comment by ann0see** [comment]
In the upcoming version 3.10.0 only Windows 10 and 11 will be supported. 
32 Bit may work on earlier versions. We still recommend to update old OS
